spec/bowling_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'bowling'
+
+describe 'A bowling game' do
+
+  before do
+    @game = Bowling.new
```

Comment:
Instead of @s and before do you can use subject { described_class.new } at the beginning and then use subject to refer to the game

Reply:
sorry if this isnt clear! we can chat this afternoon anyway